src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.tsx

```diff
@@ -254,6 +255,23 @@ export const ConversationTabs = ({
             />
           </>
         )}
+
+        <div className="conversations-sidebar-divider" />
+
+        <div className="conversations-sidebar-title" css={{marginBlock: '32px 0'}}>
+          {t('meetings.navigation.parent.label')}
+        </div>
+
+        <ConversationTab
+          title={t('meetings.navigation.title')}
+          label={t('meetings.navigation.label')}
+          type={SidebarTabs.MEETINGS}
+          Icon={<CallIcon />}
+          onChangeTab={onChangeTab}
```

Comment:
please check when a tab is selected the voice over announces its name and aria-controls is pointing to its panel. I have noticed all the tab item opens under <div id="conversations"> that might be useful.

Reply:
Thanks for pointing this out.
VO is announcing the correct information.